### PR TITLE
cpu_size_fraction as integer

### PR DESCRIPTION
**Pull request by jsydir**
Converted internal representation of cpu_size_fraction to integer to avoid errors caused by floating point arithmetic

### CODE REVIEW

File: dto.c [hunk 1]
```diff
@@ -1407,7 +1406,7 @@ static void dto_memset(void *s, int c, size_t n, int *result)
 	thr_desc.pattern = memset_pattern;
 
 	/* cpu_size_fraction guaranteed to be >= 0 and < 1 */
```
Comment (byrnedj) [inline dto.c:1408]: I would just update this comment to say the new bound (>= 0 and < 100 correct?)

File: dto.c [hunk 2]
```diff
@@ -106,7 +106,7 @@ static uint8_t use_std_lib_calls;
 static enum numa_aware is_numa_aware;
 static size_t dsa_min_size = DTO_DEFAULT_MIN_SIZE;
 static int wait_method = WAIT_YIELD;
-static double cpu_size_fraction;
+static size_t cpu_size_fraction;
```
Comment (byrnedj) [inline dto.c:109]: maybe a comment here saying the range (0 to 99?)

**Review by byrnedj — COMMENTED** [review]
Looks good, just suggested some minor comments. When those are done you can squash and merge.

File: dto.c (hunk 1 above)
Comment (jsydir) [reply dto.c]: I've added the comment

File: dto.c (hunk 2 above)
Comment (jsydir) [reply dto.c]: I've added this one also

**Pull request merged**